### ISSUE TITLE
🐛 Bug Report: User total is zero (0) in project console after minor version upgrade

### ISSUE THREAD

**Issue by wernerm** [appwrite/appwrite#5628]
### 👟 Reproduction steps

When I upgraded from version 1.3.3 to 1.3.5, the total amount of users in the "AUTH" summary block of my projects changed to zero (0). 

Before the upgrade, it showed the correct total number of users in the project.

![Screenshot_20230601_225526](https://github.com/appwrite/appwrite/assets/411270/9737cc44-6011-49c0-b70e-b702e9092e3e)

### 👍 Expected behavior

It should show the correct total number of users in the project. (It used to work fine before the update.)

### 👎 Actual Behavior

It displayed zero (0) after the update and continues to display zero (0) - this remains unchanged, even when adding or removing users from the project. The number does not change anymore after the upgrade. (It used to work fine before the update.)

### 🎲 Appwrite version

Version 1.3.x

### 💻 Operating system

Linux

### 🧱 Your Environment

This issue affects the Appwrite console.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by wernerm** [comment]
[👋](https://emojipedia.org/waving-hand/) I will have to confirm this after doing another installation and upgrade. 

I just realised that there is a small chance that I have done the upgrade after adding no more users during the preceding 30-day time period. In that specific case, the `auth` total would correctly reflect 0, since the time period for the project statistics widget is 30 days. 

So it could (of course) be a coincidence! I will try to confirm when upgrading one if our other Appwrite installations. Apologies if I just noticed a coincidence.
